week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -39,6 +59,13 @@ Write a function that:
 
 function formatPercentage(arr) {
   
+  for (let i=0; i < arr.length; i++){
+    if (arr[i] > 100){arr[i] = 100}
+
+    arr[i] = Math.ceil(arr[i]*100)/100
```

Comment:
Running tests can provide some immediate feedback here. Note that `Math.ceil` always rounds up. So from `0.372` you will actually get `0.38%` rather than `0.37%`.

Can you find a function that rounds the number to the closest integer in the way we'd expect here?